inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/configuration/Constants.java

```diff
@@ -336,4 +336,33 @@
     public static final ConfigOption<Integer> ORC_SINK_BATCH_SIZE =
             key(HIVE_SINK_PREFIX + "orc.row.batch.size")
                     .defaultValue(64);
+
+    // ------------------------------------------------------------------------
+    //  Filesystem related
+    // ------------------------------------------------------------------------
+    public enum FilesystemType {
```

Comment:
about `COS` and `CHDFS`, please add more information. what filesystem they are, and where we can get the details.